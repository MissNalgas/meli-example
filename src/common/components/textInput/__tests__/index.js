import renderer from 'react-test-renderer';
import TextInput from '..';

test('TextInput renders correctly', async () => {
	const tree = renderer.create(<TextInput/>).toJSON();
	expect(tree).toMatchSnapshot();
});
