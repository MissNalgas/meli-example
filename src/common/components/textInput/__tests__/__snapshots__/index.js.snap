// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextInput renders correctly 1`] = `
<input
  className="p-2 focus:outline-secondary h-10 w-full text-xgray"
/>
`;
